tests/plots/test_custom.py

```diff
@@ -56,3 +56,57 @@ def test_log_custom_plot_multi_y(tmp_dir):
         "x_label": "x_label",
         "y_label": "y_label",
     }
+
+
+def test_log_custom_plot_with_template_as_empty_string(tmp_dir):
+    live = Live()
+    out = tmp_dir / live.plots_dir / CustomPlot.subfolder
+
+    datapoints = [{"x": 1, "y": 2}, {"x": 3, "y": 4}]
+    live.log_plot(
+        "custom_linear",
+        datapoints,
+        x="x",
+        y="y",
+        template="",
+        title="custom_title",
+        x_label="x_label",
+        y_label="y_label",
+    )
+
+    assert json.loads((out / "custom_linear.json").read_text()) == datapoints
+    # 'template' should not be in plot_config. Default template will be assigned later.
+    assert live._plots["custom_linear"].plot_config == {
+        "title": "custom_title",
+        "x": "x",
+        "y": "y",
+        "x_label": "x_label",
+        "y_label": "y_label",
+    }
+
+
+def test_default_template_in_log_custom_plot(tmp_dir):
```

Comment:
I don't think it's worth it to add this test, has very less return. 

We can't test for every combinations.

Reply:
Since `template = ''` caused a cryptic error, I added this test as a guardrail to ensure that any `template = None` is handled explicitly. 
But on a second thought, I think you are correct. This test is for `log_plot()` which has default value for template. `CustomPlot.__init__()` doesn't have a default value for template but that shouldn't be test here.
Will rectify.